admin_guide/managing_networking.adoc

```diff
@@ -1201,3 +1201,57 @@ objects:
             name: default
 ...
 ----
+
+[[admin-guide-enabling-hsts]]
+== Enabling HTTP Strict Transport Security
+
+HTTP Strict Transport Security (HSTS) policy is a security enhancement, which
+requests that traffic to the host be HTTPS. This is useful for ensuring secure
+interactions with websites, or to offer secure application for the user's
+benefit.
+
+The default behavior is to drop any HTTP requests that require HTTPS. Enabling
+HSTS lets you set up a redirect so that the first time a client requests the
+target it is redirected to HTTPS. Any subsequent attempts will get a HTTPS
+header in the response, meaning future requests will be HTTPS.
```

Comment:
"meaning future requests will be HTTPS", the response may have a max-age=0 which drops the policy for future requests.